### DISCUSSION TITLE
Uno hosting with Azure static web app

### DISCUSSION THREAD

**Discussion by MasterThreadInf** [unoplatform/uno#10324] [Q&A] [answered]
I followed all the step as mentioned on https://platform.uno/blog/hosting-uno-platform-webassembly-apps-on-azure-static-web-apps/ .
But initial commit itself got failed and after editing .yml code I got tonnes of runtime errors as follows 

error NU3028: Package 'System.Runtime 4.3.0' from source 'https://api.nuget.org/v3/index.json': The repository primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
 error NU3037: Package 'System.Runtime 4.3.0' from source 'https://api.nuget.org/v3/index.json': The repository primary signature validity period has expired.
and many others....... 

for details :
https://github.com/MasterThreadInf/winui/actions/runs/3384257225/jobs/5621621434

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This is caused by an issue from .NET, where 5.0.103 is not supported under recent Linux distributions anymore. You can upgrade it in your workflow to be the [latest provided here](https://dotnet.microsoft.com/en-us/download/dotnet/5.0), or use the .NET 6 SDK.

The blog post you mentioned is in [our documentation](https://platform.uno/docs/articles/guides/azure-static-webapps.html), but it has the same .NET version, so we'll change that.

**Reply by MasterThreadInf** [reply]
Thanks .Its working now after updating workflow to .net 6.
